packages/xstate-wallet/src/chain.ts

```diff
@@ -324,10 +320,13 @@ export class ChainWatcher implements Chain {
       ...Transactions.createConcludePushOutcomeAndTransferAllTransaction(
         finalizationProof.flatMap(toNitroSignedState)
       ),
-      to: NITRO_ADJUDICATOR_ADDRESS
+      to: NITRO_ADJUDICATOR_ADDRESS,
+      nonce: undefined
```

Comment:
Curious, why is this needed?

Reply:
Oops, I think I that was from debugging that I forgot to remove.